### ISSUE TITLE
fix devtools url output

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2408]
currently shows object object

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.28.9](https://github.com/open-wa/wa-automate-nodejs/compare/4.28.8...4.28.9) (2022-01-08)

- Release 4.28.9 [`c69eb64`](https://github.com/open-wa/wa-automate-nodejs/commit/c69eb644c9cf3f80156e5c7ec03c65da9d3a85cd)
- 🔊 fix devtools details logging #2408 [`89556c0`](https://github.com/open-wa/wa-automate-nodejs/commit/89556c01aa7002796ed4595d89a8d52558e5e59a)
- updated types-only package [`8d98a51`](https://github.com/open-wa/wa-automate-nodejs/commit/8d98a5115830a57315ea03b6d236e2d85d81f0f8)
